.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea
+target
+.class
+.iml
```

Comment:
add RollCalculator.iml to the .gitignore. this didnt get created correctly the first time around and now we're checking in an IDE file that does not need to be used on the repository.

Reply:
Done